src/main/java/com/johnstarich/moviematcher/models/Movie.java

```diff
@@ -15,7 +15,7 @@
 	public final String rating;
 	public final String genre;
 	public final Date release_date;
-	public final String imdb_rating;
+	public String imdb_rating;
```

Comment:
We shouldn't modify imdb_rating like this. If you're trying to change the rating before it goes to the client, then we'll need to think of something else. This is bad because it makes the object mutable, and mutable objects become slow as well as confusing because we aren't sure if a method would modify the instance or return a new one.